src/EnergyPlus/VariableSpeedCoils.cc

```diff
@@ -1033,13 +1033,15 @@ namespace VariableSpeedCoils {
 
             state.dataVariableSpeedCoils->VarSpeedCoil(DXCoilNum).EvapCondPumpElecNomPower = NumArray(10);
 
-            if (state.dataVariableSpeedCoils->VarSpeedCoil(DXCoilNum).EvapCondPumpElecNomPower < 0.0) {
-                ShowSevereError(
-                    state,
-                    format("{}{}=\"{}\", invalid", RoutineName, CurrentModuleObject, state.dataVariableSpeedCoils->VarSpeedCoil(DXCoilNum).Name));
-                ShowContinueError(state, format("...{} cannot be < 0.0.", cNumericFields(10)));
-                ShowContinueError(state, format("...entered value=[{:.2T}].", NumArray(10)));
-                ErrorsFound = true;
+            if (state.dataVariableSpeedCoils->VarSpeedCoil(DXCoilNum).EvapCondPumpElecNomPower != Constant::AutoCalculate) {
```

Comment:
The Constant::AutoSize and Constant::AutoCalculate do have the same value but they mean different things. We should probably change one of the values. The        \autocalculatable

Reply:
Ah, that's useful to know - the rest of the function is peppered with `Autocalculate`s where "Autosize" is used in the IDF (e.g. Rated Air Flow Rate) - I just copied and pasted. So I imagine those were intended to be `Autosize` too?

Reply:
OK, this is correct and the idd shows autosize when it should be autocalculatable. You know an input is auto calculated when there is a rule of thumb used to size the input.

            //     Auto size high speed evap condenser pump power to Total Capacity * 0.004266 w/w (15 w/ton)
            EvapCondPumpElecNomPowerDes = varSpeedCoil.RatedCapCoolTotal * 0.004266;